frontend/device/devicelistener.lua

```diff
@@ -51,24 +49,38 @@ end
 function DeviceListener:onShowWarmth(value)
     local powerd = Device:getPowerDevice()
     if powerd.fl_warmth ~= nil then
-        UIManager:show(Notification:new{
-            text = T(_("Warmth set to %1."), powerd.fl_warmth),
-            timeout = 1.0,
-        })
+        if powerd.fl_warmth_max == nil then
+            UIManager:show(Notification:new{
+                text = T(_("Warmth set to %1."), math.floor(powerd.fl_warmth)),
+                timeout = 1.0,
+            })
+        else
+            UIManager:show(Notification:new{
+                text = T(_("Warmth set to %1."), math.floor(powerd.fl_warmth/100*powerd.fl_warmth_max)),
```

Comment:
`math.floor(powerd.fl_warmth/100*powerd.fl_warmth_max))` is difficult to understand without making obvious that fl_warmth and fl_warmth_max are using different scales.

Reply:
Would a comment be enough for clarification?
```
-- powerd.fl_warmth holds the warmth-value in the internal koreader scale [0,100]
-- powerd.fl_warmth_max is the maximum value the hardware accepts
```